File: V1_objects/ark__99999_fk4805c32z.py
from math import exp
import math


def interpretLungCancerRisk(info):
    risk = float(info['risk'])

    output = ""

    if risk > 1.35:
        output = """
        <div class="interpretation">
        <p style="font-weight: bold;">RECOMMENDATION: Consider lung cancer screening.</p>
        <p>Because the chance is <b>greater than 1.35%</b>, engaging in a process of shared
        decision making about lung cancer screening should be considered, during which other
        factors, including the health status and disposition of the individual, should be
        taken into account.</p>
        </div>
        """
    else:
        output = """
        <div class="interpretation">
        <p style="font-weight: bold;">RECOMMENDATION: Lung cancer screening is not advised.</p>
        <p>Because this chance is <b>lower than 1.35%</b>, lung cancer screening is not indicated
        as the risk of lung cancer screening exceeds its likely benefits.</p>
        </div>
        """

    return output

File: V1_objects/ark__99999_fk4805c32z.js

function interpretLungCancerRisk (info)
{


 var risk = float (info['4']);

var output = "";

 if (risk > 1.35)

 {
output= [];


console.log('RECOMMENDATION: Consider lung cancer screening. Because the chance is greater than 1.35%, engaging in a process of shared decision making about lung cancer screening should be considered, during which other factors, including the health status and disposition of the individual, should be taken into account.');

 }

 else

 {

 output= [];

console.log ('RECOMMENDATION: Lung cancer screening is not advised. Because this chance is <b>lower than 1.35% lung cancer screening is not indicated as the risk of lung cancer screening exceeds its likely benefits.');


 }

  return output;

}
